src/Panel/Panel.d.ts

```diff
@@ -0,0 +1,37 @@
+import { PureComponent, ReactNode } from 'react';
+
+export namespace PanelProps {
+
+  type HeaderSize = 'tiny' | 'small' | 'medium';
+
+  type CollapseDirection = 'up' | 'left' | 'down' | 'right';
+
+  type Header = {
+    collapse?: {
+      direction: CollapseDirection,
+      onExpand: () => void,
+      onCollapse: () => void,
+      isCollapsed?: boolean
+    },
+    size?: HeaderSize,
+    content?: any,
+    title?: ReactNode | string, // TODO(typo): wtf
+    hideTitleWhenExpanded?: boolean,
+    menu?: any
+  }
+}
+
+export type PanelProps = {
+  children: ReactNode,
```

Comment:
usually we use `any` for `children` but honestly I don't know exactly the reason...
@giogonzo is `ReactNode` ok?